mitmproxy/builtins/replace.py

```diff
@@ -29,16 +31,30 @@ def configure(self, options, updated):
                 raise exceptions.OptionsError(
                     "Invalid regular expression: %s - %s" % (rex, str(e))
                 )
-            lst.append((rex, s, cpatt))
+            if s[0:7] == "file://":
+                file_repl = True
+                if not os.path.isfile(s[7:]):
```

Comment:
Should we use https://docs.python.org/3/library/urllib.parse.html#module-urllib.parse to properly parse file urls?
Might be safer...